components/ai_chat/resources/page/components/conversation_list/index.tsx

```diff
@@ -149,20 +166,22 @@ function ConversationList(props: ConversationListProps) {
                     </div>
                     <span>{isHuman ? 'You' : 'Leo'}</span>
                   </div>
-                  {isAIAssistant && (
-                    <div className={styles.turnActions}>
-                      <CopyButton onClick={handleCopyText} />
-                      {context.currentModel?.options.leoModelOptions && (
-                        <ContextMenuAssistant
-                          ref={portalRefs}
-                          turnId={id}
-                          isOpen={activeMenuId === id}
-                          onClick={() => showAssistantMenu(id)}
-                          onClose={hideAssistantMenu}
-                        />
-                      )}
-                    </div>
-                  )}
+                  <div className={styles.turnActions}>
+                    <CopyButton onClick={handleCopyText} />
```

Comment:
you can copy human text now? i thought copy was only allowed on assitant turns.

Reply:
<img width="540" alt="Screenshot 2024-07-08 at 9 32 53 PM" src="https://github.com/brave/brave-core/assets/4730197/3fe7b91b-c82b-4a57-88d5-b961dc2d5c0c">
I think so as it's how it looks in the figma.